### ISSUE TITLE
'Stop building on the first error" & database projects cause VS 2015 RC to hang

### ISSUE THREAD

**Issue by DotNetSparky** [StefanKert/BuildVision#10]
With BuildVision enabled, and the "Stop building on the first error" option enabled, if there is a build error when building a database project, then VS hangs/locks up (completely unresponsive, idle CPU... "deadlock" type behaviour)... have to ultimately end-task VS.

In my specific case, it was a typo in a post-deploy script.  The hang would occur right after `Writing model to [project-path]\obj\db\Model.xml...` was written to the console, but the build error would never be written (it took me awhile to finally figure out there even WAS a build error :-)).

With 'stop on first build error' enabled:

```
------ Build started: Project: MyDatabase, Configuration: db Any CPU ------
    Loading project references...
    Loading project files...
    Building the project model and resolving object interdependencies...
    Validating the project model...
    Writing model to C:\Projects\TestProject\MyDatabase\obj\db\Model.xml...
```

VS has completely frozen at this point--clicking in any VS window causes Windows to fade the window.  I've let it sit for over an hour and it was still locked up (and cpu completely idle--I suspect deadlocked on something?)

If the "Stop building on first error" option is disabled, then when the build fails the error shows in the Output window, right after the previous "Writing model..." line

```
------ Build started: Project: MyDatabase, Configuration: db Any CPU ------
    Loading project references...
    Loading project files...
    Building the project model and resolving object interdependencies...
    Validating the project model...
    Writing model to C:\Projects\TestProject\MyDatabase\obj\db\Model.xml...
C:\PROJECTS\TESTPROJECT\MYDATABASE\SCRIPTS\POSTDEPLOYMENT\SETEXTPROP.SQL(45,88): Error:  SQL72007: The syntax check failed 'Incorrect syntax near $(MyVar).' in the batch near:
    '   exec sp_updateextendedproperty @name = N'MyVar', @value = $(MyVar), @level0type = N'SCHEMA', @level0name = N'dbo';'
Done building project "MyDatabase.sqlproj" -- FAILED.

Build FAILED.
========== Build: 0 succeeded or up-to-date, 1 failed, 0 skipped ==========
```

I should also note it seems to be ANY sort of error while building the database project, failed references, etc., all seem to freeze.

This is in Visual Studio 2015 RC (Community edition), under Windows 8 Pro.  I haven't tried to reproduce this under VS 2013 (or any other VS versions), nor have I tried changing any other settings in BuildVision.  I also didn't try disabling all the other extensions to see if it's caused by a conflict (other than ReSharper and WebEssentials, so I can rule those out).  I couldn't try with starting devenv.exe in /SafeMode because my database projects then wouldn't load.

**Comment by nagits** [comment]
Hi @DotNetSparky, 
thanks for reporting this!

You have described in great detail the issue, thanks :+1:!

**Comment by StefanKert** [comment]
@JoshuaGarrison27 fixed this issue in #50. You can grab the new version under [https://ci.appveyor.com/project/StefanKert/buildvision/branch/master/artifacts](https://ci.appveyor.com/project/StefanKert/buildvision/branch/master/artifacts) or you wait for the release :). Please let me know if this fixed your issue. I am closing this now. If this issue still occurs please reopen.
